cmd/audius-ctl/main.go

```diff
@@ -13,19 +13,19 @@ var (
 )
 
 func main() {
-	rootCmd := &cobra.Command{
+	var rootCmd = &cobra.Command{
 		Use:   "audius-ctl [command]",
 		Short: "CLI for provisioning and interacting with audius nodes",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		Run: func(cmd *cobra.Command, args []string) {
 			if displayVersion {
 				logger.Out(Version)
-			} else {
-				upCmd.RunE(cmd, args)
+				return
 			}
-			return nil
+			cmd.Help()
```

Comment:
I'm going to take a few min to get to the bottom of this error just in case it breaks any assumptions I have about how cobra is supposed to work

Reply:
Did not get a segfault when running it without commands, but still interested in reproducing this bug. What context did you have active when you ran it?

Also, I'm okay with this change so don't consider this a blocker, just interested in getting to the bottom of the segfault since it's completely unexpected behavior.